src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -61,6 +61,11 @@
      */
     private int directionId = -1;
```

Comment:
This is a comment for lines 49-51. I noticed this weirdness in the javadoc:

https://github.com/ibi-group/OpenTripPlanner/blob/946da1d32cfbe87deb8476fc127266190a234d05/src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java#L49-L52

Reply:
ok, that's a good bit more context. It turns out there aren't any codepaths to set this anyways, so I went ahead and removed it altogether in e2bb4b8b2ab2c9b846496dcc76ae0a039ca59102